### PR TITLE
Support for fetching mail overview

### PR DESCRIPTION
**Pull request by jkuchar**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "files": ["MailLibrary/loader.php"]
     },
     "require": {
-        "php": ">= 5.3.0",
+        "php": ">= 7.1.0",
```
Comment (jkuchar) [inline composer.json]: Is useful because of `??` operator

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - composer self-update
+
+install:
+  - composer install --no-interaction --prefer-source
+
+script:
+  - ./vendor/bin/tester -c ./tests/php-unix.ini -s ./tests/
+
+after_failure:
+  # Vytiskne obsah souborů *.actual
```
Comment (kerleba) [inline .travis.yml]: this should be in english

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - for i in $(find ./tests -name \*.actual); do echo "--- $i"; cat $i; echo; echo; done
```
Comment (kerleba) [inline .travis.yml]: Missing new line at the end of file

File: MailLibrary/Drivers/ImapDriver.php [hunk 4]
```diff
@@ -529,18 +521,14 @@ public function uploadRawMessage($contentOfMessage)
```
Comment (kerleba) [inline MailLibrary/Drivers/ImapDriver.php:521]: in `uploadRawMessage`: 4th parameter of `imap_append` is string so `$options` must be string.

**Review by kerleba — CHANGES_REQUESTED** [review]

**Comment by kerleba** [comment]
With fixed `uploadRawMessage` it is working perfectly.

File: MailLibrary/Drivers/ImapDriver.php (hunk 4 above)
Comment (jkuchar) [reply MailLibrary/Drivers/ImapDriver.php:521]: fixed

**Comment by jkuchar** [comment]
@kerleba ready for more comments

**Comment by jkuchar** [comment]
@kerleba could I merge this? Any more comments?

**Review by kerleba — APPROVED** [review]

**Comment by kerleba** [comment]
@jkuchar you can merge it

**Comment by jkuchar** [comment]
Thanks for review!

**Pull request merged**